docs/pages/build-reference/e2e-tests.md

```diff
@@ -357,3 +357,101 @@ Run a build with `eas build -p ios --profile test` and wait for the build to fin
 ## Repository
 
 The full example from this guide is available at https://github.com/expo/eas-tests-example.
+
+## Alternative Approaches
+
+### Using development builds to speed up test run time
+
+The above guide explains how to run E2E tests against a release build of your project, which requires executing a full native build before each test run. Re-building the native app each time you run E2E tests may not be desirable if only the project JavaScript or assets have changed, but this is necessary for release builds because the app JavaScript bundle is embedded into the binary.
+
+We can use [development builds](/development/introduction/) to load from a local development server or from [published updates](/eas-update/introduction/) instead, and save time and CI resources. This can be done by having your E2E test runner invoke the app with a URL that points to a specific update bundle URL, as described in the [development builds deep linking URLs guide](/development/development-workflows/#deep-linking-urls).
+
+Development builds typically display an onboarding welcome screen when an app is launched for the first time — this is intended to provide context about the expo-dev-client UI for developers, but it can interfere with your E2E tests (which expect to interact with your app and not an onboarding screen). In order to skip the onboarding screen in a test environment, the query parameter `disableOnboarding=1` can be appended to the project URL (an EAS Update URL or a local development server URL).
+
+An example of such a Detox test is shown below. Full example code is available on the [eas-tests-example repository](https://github.com/expo/eas-tests-example).
+
+<Collapsible summary="homeScreen.e2e.js">
+
+```js
+const {
+  sleepAsync,
+  getConfigurationName,
+  // getDevLauncherPackagerUrl,
+  getLatestUpdateUrl,
+  getDeepLinkUrl,
+} = require('./utils');
+
+describe('Home screen', () => {
+  beforeEach(async () => {
+    await device.launchApp({
+      newInstance: true,
+    });
+    if (getConfigurationName().indexOf('debug') !== -1) {
+      await device.openURL({
+        // Local testing with packager
+        //url: getDeepLinkUrl(getDevLauncherPackagerUrl(platform)),
+        // Testing latest published EAS update for the test_debug channel
+        url: getDeepLinkUrl(getLatestUpdateUrl()),
+      });
+    }
+    await sleepAsync(3000);
+  });
+
+  it('"Click me" button should be visible', async () => {
+    await expect(element(by.id('click-me-button'))).toBeVisible();
+  });
+
+  it('shows "Hi!" after tapping "Click me"', async () => {
+    await element(by.id('click-me-button')).tap();
+    await expect(element(by.text('Hi!'))).toBeVisible();
+  });
+});
+```
+
+</Collapsible>
+
+<Collapsible summary="utils.js">
+
+```js
+const { exec } = require('child_process');
+const argparse = require('detox/src/utils/argparse');
+const appConfig = require('../app.json');
+
+//////////// General util methods //////////////
```

Comment:
```suggestion
/**
 * General utility methods
 */
```

this is the preferred style in these types of section-marking comments. might require making the changes in the other repo to match

Reply:
Fixed :)